blackbox/docs/config/logging.rst

```diff
@@ -40,114 +63,132 @@ configuration.
 And here is a snippet of the generated properties ready for use with log4j.
 You get the point.
 
-.. code-block:: yaml
+.. SEEALSO::
 
-  log4j.rootLogger=INFO, console
+    Consult the `PropertyConfigurator`_ documentation or the `configuration
+    section`_ of the Log4j documentation for more information.
 
-  log4j.logger.action=DEBUG
+.. _conf-logging-log4j-loggers:
 
-  log4j.appender.console=org.elasticsearch.common.logging.log4j.ConsoleAppender
-  log4j.appender.console.layout=org.apache.log4j.PatternLayout
-  log4j.appender.console.layout.conversionPattern=[%d{ISO8601}][%-5p][%-25c] %m%n
+Loggers
+.......
 
-   ...
+.. FIXME: need a full list of available loggers
```

Comment:
@mxm can you help me with this?

if you provide me with a list of names and a short description of each, I will tabulate this and push a fixup

Reply:
The idea here is that you can write your own logger and specify it here. I wouldn't recommend any other than the default ConsoleAppender or the ones which are included in Log4j2.

Reply:
hmm. no I mean, for example, "org.crate.action.sql". is this the only "action" available or are they more? perhaps I am mixing up my terminology here!

Reply:
"org.crate.action.sql" here is the name of the logger "action".

Reply:
@mxm so there's only one value that this action can ever be? I figured we might have org.crate.action.foo or whatever

Reply:
"action" is the identifier of the logger which needs to be unique. Additionally, you can give it a name.